validator.py

```diff
@@ -36,3 +37,25 @@
 else:
   print("Weak")
 
+
+
+# Test code with a password that should be considered 'Strong'
+test_password = "Hello@$World19"
+
+num_count = 0
+char_count = 0
+char = ["!", "@", "#", "$", "%", "&", "*"]
+nums = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
+
+for var in test_password:
+    if var in char:
+        char_count += 1
+    if var in nums:
+        num_count += 1
+
+if (char_count > 1 and num_count > 1) and (len(test_password) > 6):
+    print("Strong")
+else:
+    print("Weak")
```

Comment:
**Suggestion:** Wrap the password strength checking logic into a function for better reusability and readability. Also, the logic for checking if a character is a number can be simplified by using the built-in isdigit() function.
```suggestion
def check_password_strength(password):
    num_count = 0
    char_count = 0
    special_chars = ["!", "@", "#", "$", "%", "&", "*"]

    for char in password:
        if char in special_chars:
            char_count += 1
        if char.isdigit():
            num_count += 1

    if (char_count > 1 and num_count > 1) and (len(password) > 6):
        return "Strong"
    else:
        return "Weak"

test_password = "Hello@$World19"
print(check_password_strength(test_password))
```